bHY6yh-d28342/cod-yyyiou_bHY6yh-d28342.xml

```diff
@@ -0,0 +1,133 @@
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-08-05" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="37-v"/>
+            <cb ed="#C" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d28342"><!-- p2l3-r3 -->
+            <head xml:id="bHY6yh-d28342-Hd1e108">Rubrica 3</head>
+            <head xml:id="bHY6yh-d28342-Hd1e111" type="question-title">De prediectione suiiipsius</head>
+            <p xml:id="bHY6yh-d28342-d1e114">
+              De dilectione
+              <lb ed="#C"/>sui ipsius
+            </p>
+            <!--next paragraph begins on the same line as the previous paragraph-->
+            <p xml:id="bHY6yh-d28342-d1e119">
+              <ref xml:id="bHY6yh-d28342-Rd1e115">Salomon in prouerbiorum
+              <lb ed="#C"/>capitulo V</ref> <quote xml:id="bHY6yh-d28342-Qd1e125">non des ali<lb ed="#C" break="no"/>enis 
+              honorem tuum</quote> 
+            </p>
+            <p xml:id="bHY6yh-d28342-d1e131">
+              ¶ <ref xml:id="bHY6yh-d28342-Rd1e129">ysaias C35</ref> <quote xml:id="bHY6yh-d28342-Qd1e135">Cum uideris
+              <lb ed="#C"/>nudum operi eum et carnem tuam ne despexeris</quote>
+            </p>
+            <p xml:id="bHY6yh-d28342-d1e144">
+              <lb ed="#C"/>¶ <ref xml:id="bHY6yh-d28342-Rd1e144">yesus filius cidrach in ecclesiastico capitulo 33</ref> <quote xml:id="bHY6yh-d28342-Qd1e148">filio et mulieri
+              <lb ed="#C"/>fratri et amico non des potestatem super te in ui<lb ed="#C" break="no"/>ta 
+              tua non dederis alienis possessiones tuas
+              <lb ed="#C"/>ne forte peniteat te et depreceris pro illis dum
+              <lb ed="#C"/>adhuc superes et aspiras non mutabit
+              <lb ed="#C"/>te omnis caro melius est enim ut filii tui te rogent
+              <lb ed="#C"/>quam te respicere in manus filiorum tuorum</quote>
+              <pb ed="#C" n="38-r"/>
+              <cb ed="#C" n="a"/>
+            </p>
+            <p xml:id="bHY6yh-d28342-d1e167">
+              <lb ed="#C"/>¶ <ref xml:id="bHY6yh-d28342-Rd1e169">Seneca de beneficiis libro 2</ref> <quote xml:id="bHY6yh-d28342-Qd1e171">Dabo egenti uide
+              <lb ed="#C"/>summa</quote>
```

Comment:
remember pb and cb should never end a paragraph but always start the paragraph, like so...
```suggestion
              <lb ed="#C"/>quam te respicere in manus filiorum tuorum</quote>
            </p>
            <p xml:id="bHY6yh-d28342-d1e167">
              <pb ed="#C" n="38-r"/>
              <cb ed="#C" n="a"/>
              <lb ed="#C"/>¶ <ref xml:id="bHY6yh-d28342-Rd1e169">Seneca de beneficiis libro 2</ref> <quote xml:id="bHY6yh-d28342-Qd1e171">Dabo egenti uide
              <lb ed="#C"/>summa</quote>
```